Command/GraphvizGenerateCommand.php

```diff
@@ -43,7 +43,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $dest = $this->getApplication()->getKernel()->getRootDir() . '/propel/graph/';
+        $dest = $this->getApplication()->getKernel()->getProjectDir() . '/var/propel/graph/';
```

Comment:
Is this change in location intentional?
`./app/propel/graph` --> `./var/propel/graph`

Reply:
It is, seemed better to me to group generated files together in a subdir than in the root dir.